aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -36,15 +36,18 @@ static CPUCapability compute_cpu_capability() {
     }
 #elif defined(HAVE_SVE_CPU_DEFINITION)
     int sve_vl = cpuinfo_get_max_arm_sve_length(); //Returns maximum SVE VL supported by your HW.
-#ifdef HAVE_SVE256_CPU_DEFINITION
-    if (strcmp(envar, "sve256") == 0) {
```

Comment:
should we continue to support the sve256 setting so that people don't get broken by the change?

Reply:
This is an runtime option which user can specify in the application to change the backend at runtime.

**Here's the code:**
import os
os.environ["ATEN_CPU_CAPABILITY"] = "sve"

Our understanding is that, most of the developers/users when running PyTorch on ARM machines are not really aware if their hardware is SVE VL=128/256/512. so, ideally, it is easy for them to pass just **"sve"** and we have an internal logic in the code which checks VL of the hardware and assigns the right backend to the user.

![image](https://github.com/user-attachments/assets/8195da86-9906-4df5-996f-fc17989347ab)